redis-pq-demo/README.md

```diff
@@ -18,20 +18,20 @@ After establishing a connection with the mediator. You can inspect the contents
 
 ## Example agents
 
-If you want to run a set of agents using the mediator, consider the acapy playground demo. 
+If you want to run a set of agents using the mediator, consider the acapy [playground](https://github.com/hyperledger/aries-cloudagent-python/tree/f7db243430285edeb721f079ff0694bb76f41a3e/demo/playground) demo. 
```

Comment:
Necessary to tie it to a commit?  Creates technical debt that would be nice to avoid, if possible.

Reply:
will update, good catch!

Reply:
It’s there twice :-)

Reply:
cleaned up with a permalink.

Reply:
I’m confused.  Why are you referencing a commit vs. just referencing the `main` branch?  That’s what I was wondering.  Wouldn’t it be safer to just reference `main`?

Reply:
my thinking was that these two projects get out of sync. so at least this sends them to the instructions as they existed when this was updated.  however, now that you make me think of it, if the aca-py playground changes then those "old" instructions could be more confusing!... ok will change to point at main.